packages/core/src/theming/prebuilt-themes/qcca/components/scrollbar.scss

```
@@ -0,0 +1,17 @@
+@use 'sass:map';
+@use '../base';
+
+*::-webkit-scrollbar {
+    width: 6px;
+}
+
+*::-webkit-scrollbar-track {
+    box-shadow: inset 0 0 5px #dae6f0;
```

Il faudrait utiliser la palette de couleur ou aller chercher directement dans les coleurs officiel du QCCA
```suggestion
    box-shadow: inset 0 0 5px base.$blue-light;
```